Lecture3/Task2/Application.java

```diff
@@ -0,0 +1,63 @@
+import java.util.*;
+
+public class Application {
+    public static void main(String[] args) {
+
+        List<Student> studentList = new ArrayList<>();
+
+        studentList.add(new Student(1, "Cristiano", "Ronaldo", 33));
+        studentList.add(new Student(2, "Roberto", "Carlos", 40));
+        studentList.add(new Student(3, "Vladimir", "Vysotsky", 23));
+        studentList.add(new Student(4, "Paolo", "Maldini", 19));
+        studentList.add(new Student(5, "Andrei", "Ivanov", 22));
+        studentList.add(new Student(6, "Sergey", "Petrov", 24));
+        studentList.add(new Student(7, "Alexey", "Poluyan", 25));
+        studentList.add(new Student(8, "Yuriy", "Gagarin", 28));
+        studentList.add(new Student(9, "Olga", "Buzova", 33));
+        studentList.add(new Student(10, "Merylin", "Monro", 24));
+
+        Collections.sort(studentList, new AgeComparator());
+        System.out.println("List of students sorted by age:");
+        for(int i = 0; i < studentList.size(); i++){
+            System.out.println(studentList.get(i).getId() + " " + studentList.get(i).getName() + " " +
+                    studentList.get(i).getSurname() + " " + studentList.get(i).getAge());
+        }
+
+        double sumAge = 0;
+        for(int i = 0; i < studentList.size(); i++){
```

Comment:
try to avoid indexes when they don't need by logic
for (Student aStudentList : studentList) {
            sumAge += aStudentList.getAge();
        }